Resources/public/js/views/services/ez-serversideviewservice.js

```diff
@@ -161,6 +166,32 @@ YUI.add('ez-serversideviewservice', function (Y) {
         },
 
         /**
+         * Fires notify event basing on node.
+         * notify event does not bubble to app so triggering it directly on app instance
```

Comment:
it's an implementation details so I would put that inline right above the `app.fire` call. And the text should something like:

```js
// the app is not yet a bubble target of the view service,
// so we are using the app to fire the event
// see https://jira.ez.no/browse/EZP-23013
```